blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionHandler.java

```diff
@@ -99,19 +99,32 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
 
             requestCount.inc();
             final String tenantId = request.getHeader("tenantId");
-            JSONMetricsContainer jsonMetricsContainer = null;
+            JSONMetricsContainer jsonMetricsContainer;
+            List<Metric> metrics;
+
             final Timer.Context jsonTimerContext = jsonTimer.time();
 
             final String body = request.getContent().toString(Constants.DEFAULT_CHARSET);
             try {
                 jsonMetricsContainer = createContainer(body, tenantId);
 
                 if (jsonMetricsContainer == null) {
-                    log.warn(ctx.getChannel().getRemoteAddress() + " No valid metrics");
+                    log.warn(ctx.getChannel().getRemoteAddress() + " No valid json content.");
                     DefaultHandler.sendResponse(ctx, request, "No valid metrics", HttpResponseStatus.BAD_REQUEST);
```

Comment:
If you are changing the log.warn(), why don't we change the response as well, to match?